shared/js/l10n.js

```diff
@@ -1932,13 +1942,20 @@
       return;
     }
 
-    var l10n = getL10nAttributes(element);
+    var l10nId = element.getAttribute('data-l10n-id');
 
-    if (!l10n.id) {
+    if (!l10nId) {
       return false;
     }
 
-    var entity = this.ctx.getEntity(l10n.id, l10n.args);
+    var l10nArgs = element.getAttribute('data-l10n-args');
+
+    var entity = l10nArgs ?
+      this.ctx.getEntity(
+        l10nId,
+        JSON.parse(l10nArgs.replace(reHtml, escapeL10nArgs))
```

Comment:
Could you move the `l10nArgs ?` check into the second argument of `getEntity`?  It can be `undefined` no problem.